### DISCUSSION TITLE
Tutorial for beginners?

### DISCUSSION THREAD

**Discussion by itrack** [coqui-ai/Trainer#82] [Q&A]
Hello,
I don't have much experience but I would like to train a model, can you help me with some simplified explanations?
Thank you!

**Comment by erogol** [comment]
Train a model or a TTS model?

**Reply by itrack** [reply]
STT model

**Reply by erogol** [reply]
if you mean 🐸STT, it does not use 👟Trainer. But if you want to use 👟Trainer for your model, then you can take https://github.com/coqui-ai/Trainer/blob/main/tests/test_train_mnist.py as an example. You can also check 🐸TTS models. 

Sorry, we don't have extensive docs currently.
